### ISSUE TITLE
RAW: wrong default value for use_camera_matrix?

### ISSUE THREAD

**Issue by devernay** [OpenImageIO/oiio#1628]
in rawinput.cpp, oiio sets use_camera_matrix to 0 by default, but the libraw default should be 1 (which means "use the camera matrix for DNG only").
ref: http://www.libraw.org/docs/API-datastruct-eng.html#libraw_output_params_t

**Comment by devernay** [comment]
another bug:
```
            m_spec.erase_attribute("raw:Exposure", TypeDesc::STRING);
```
should be `TypeDesc::FLOAT`

**Comment by lgritz** [comment]
Right. Fix coming up.

**Comment by lgritz** [comment]
Proposed fix: #1629
